Adding instructor note about piloting to learn (#141)
carpentries/lesson-development-training: instructors/instructor-notes.md

title: FIXME
---

### Training Themes

#### Lesson Development as a Learning Process

A recurring theme in this training is the discussion of the benefits of piloting a new 
lesson and treating teaching as an opportunity to learn and improve the lesson. 

> You should try to frame the discussion this way and to draw on your own experience 
> when talking about the ways in which piloting a lesson can help you improve it.

Reviewer: ```suggestion
We have found it helpful to frame the discussion this way and to draw on our own experience 
when talking about the ways in which piloting a lesson can help to improve it.
```

Author: Good idea!

For example, when teaching the assessments or narrative section you might discuss a time 

> when you piloted a workshop and realize there was a concept missing in the demo/narrative that the learners needed in order to be able to complete the exercise.

Reviewer: ```suggestion
when you piloted a workshop and realized there was a concept missing in the demo/narrative
that learners needed in order to be able to complete the exercise.
```
